spec/system/placements/edit_spec.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe "placements/edit", type: :system do
+  let(:now) { Date.new(2025, 1, 2) }
+  let(:casa_org) { create(:casa_org, :with_placement_types) }
+  let(:admin) { create(:casa_admin, casa_org:) }
+  let(:casa_case) { create(:casa_case, casa_org:, case_number: "123") }
+  let(:placement_type) { create(:placement_type, name: "Reunification", casa_org:) }
+  let(:placement) { create(:placement, placement_started_at: "2024-08-15 20:40:44 UTC", casa_case:, placement_type:) }
+
+  before do
+    travel_to now
+    sign_in admin
+    visit casa_case_placement_path(casa_case, placement)
+    click_link("Edit")
```

Comment:
```suggestion
    visit edit_casa_case_placement_path(casa_case, placement)
```

I think would work - can run `bin/rails routes -g casa_case_placement` to check it's there. no big deal, just on less page load for every test.

Reply:
I would argue that those are two different tests. Navigating to the index first catches any issues with the edit link not being present etc.

That being said that might already be covered in another test and be unnecessary.

Reply:
yes, but I would say that is a concern of the index system or view spec... `it "displays link to edit" do` or similar there... edit spec shouldn't worry about index (or the other ways we can get to this page) at all imo.